rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java

```diff
@@ -126,6 +127,26 @@ JobId submit(URL job)
      * precious result.
      * <p>
      *
+     * @param job       a job provided as a url
+     * @param variables job variables to use during the job execution
```

Comment:
please add the headerParams in the javadoc

Reply:
Good catch, fixed